lib/from_erlang.ml

```diff
@@ -20,8 +20,8 @@ let expr_of_literal = function
   | F.LitString (_line_t, s) ->
      (* string is a list of chars in Erlang *)
      let l = String.to_list_rev s in
-     List.fold_left l ~init:ListNil ~f:(fun acc c -> ListCons (Constant (Number (Char.to_int c)), acc))
-  | l -> Constant (const_of_literal l)
+     List.fold_left l ~init:ListNil ~f:(fun acc c -> ListCons (Constant (-1, Number (Char.to_int c)), acc))
+  | l -> Constant (-1, const_of_literal l)
```

Comment:
Because `l` has line information, how about making `const_of_literal` return line number and `let (line, c) = const_of_literal l in Constant (line, c)`?

Reply:
I fixed that point. Thanks.